tests/phpunit/CRM/Contribute/Form/Contribution/ConfirmTest.php

```diff
@@ -239,6 +207,27 @@ public function testSeparatePaymentConfirm(): void {
     $this->assertEquals(1, $financialTrxn['card_type_id']);
     $assignedVariables = $form->getTemplateVariables();
     $this->assertTrue($assignedVariables['is_separate_payment']);
+    // Two emails were sent - check both. The first is a contribution
+    // online receipt & the second is the membership online receipt.
+    $this->assertMailSentContainingStrings([
```

Comment:
@eileenmcnaughton how do we know that it isn't just testing the one email content how does it check each separately?

Reply:
@seamuslee001 the second parameter to that function is an index